spec/client_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+describe ActionNetworkRest::Client do
+  let(:api_key) { 'abcde12345' }
+
+  it 'should turn an api_key option into an additional header' do
+    client = ActionNetworkRest::Client.new(api_key: api_key)
+    expect(client.default_options[:additional_headers]).to eq({'OSDI-API-Token' => api_key})
+  end
```

Comment:
Should also spec that the person/people method here?

Reply:
It's effectively covered in the `person` specs, but I'm not against being explicit about it here.